### ISSUE TITLE
Support multiple authors

### ISSUE THREAD

**Issue by ayberkt** [ayberkt/chalmers-msc-thesis-template#1]

**Comment by Jassob** [comment]
I'll volunteer to look into this issue as I probably need it for our report.

**Comment by ayberkt** [comment]
> I'll volunteer to look into this issue as I probably need it for our report.

Great!

**Comment by ayberkt** [comment]
@Jassob I suppose the ideal way to input multiple author names would follow the LaTeX convention of

```
\author{First Author \and Second Author}
```

**Comment by Jassob** [comment]
Yes and that is now supported by the template.
I think moving to using the titling macros rather than `\@author` and then removing `\makeatletter`/`\makeatother` fixed the original issue where `\and` previously caused some error.

**Comment by Jassob** [comment]
However it would be nice if we could style the front page in the same way as is done in previous reports.
An example of how authors are styled can be seen in [this example](https://odr.chalmers.se/bitstream/20.500.12380/300609/1/CSE%2019-114%20Lawrence%20Sebastian.pdf).

**Comment by ayberkt** [comment]
> However it would be nice if we could style the front page in the same way as is done in previous reports.

Just changing the delimiter from `,` to line break should mostly fix it, with the exception of the acknowledgements page.

**Comment by Jassob** [comment]
> Just changing the delimiter from `,` to line break should mostly fix it, with the exception of the acknowledgements page.

The `,` was added by the `\and` command, do you know if there are any way to override its interpretation in different environments? If you don't I will look into it, because I think that it should be doable :)

Aside: Setting `\author{John Doe \\ John Bloggs}` works in many cases, but the imprint page and the acknowledgements are still a bit off, because they assume that author will be a single line.

**Comment by ayberkt** [comment]
> do you know if there are any way to override its interpretation in different environments?

No, unfortunately.

Maybe we should simply ask for two author names and do a simple if-def on the second one. Here is an example of these kinds of conditionals in TeX: https://tex.stackexchange.com/questions/33576/conditional-typesetting-build.

**Comment by ayberkt** [comment]
I would have preferred `\and`-separation but it seems like asking for multiple author names will be much simpler than parsing that or otherwise manipulating the definition of `\and`.

**Comment by ayberkt** [comment]
@Jassob I lost track of the progress on this issue. Can we consider it done?

**Comment by Jassob** [comment]
No, I don't think so. It still does not do anything sensible by either \and or \\-separation.

I have not really been paying attention to this issue however..

**Comment by ayberkt** [comment]
Ah okay. I thought it might have been completed as you are using this in your thesis.
